Components/Task.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { View, TouchableHighlight, Text } from 'react-native';
+import { CheckBox } from 'react-native-elements';
+
+export default class Task extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isDone: false,
+            textStyle: {
+                color: '#000',
+                textDecorationLine: 'none',
+            },
+        }
+    }
+
+    deleteTask = () => {
+        this.props.onDelete(this.props.taskNumber);
+    }
+
+    onDone = () => {
+        let done = !this.state.isDone;
+        this.state.isDone = !this.state.isDone;
```

Comment:
remove style logic from this method. after this it will look like  this:
 onDone = () => {
  this.setState({ isDone: !this.state.done});
}